<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateBidPopup</class>
 <widget class="QDialog" name="UpdateBidPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="update">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Update </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Bid id</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Bid rate (RUB to USD)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Bid quantity</string>
   </property>
  </widget>
  <widget class="QLabel" name="bid_id">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="bid_rate">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="bid_quantity">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
